pyhoma/client.py

```diff
@@ -294,6 +295,8 @@ async def check_response(response: ClientResponse) -> None:
             result = await response.json(content_type=None)
         except JSONDecodeError:
             result = await response.text()
+            if "maintenance" in result:
+                raise MaintenanceException("Server is down for maintenance")
```

Comment:
Could we possibly also check for 503 status code? At least Somfy is using the correct status code this time. `503 Service Unavailable.`

Reply:
I don't think we have to check the status code again. We are already in the error case.

Reply:
I am fine with the current implementation. We could also do it different, like this:

catch 503 error
 if 503 and "maintenance" -> throw maintenance
 otherwise throw ServiceUnavailableException

Hopefully we don't need the last one, but otherwise we throw a good exception at least.